src/components/Card/Card.test.tsx

```diff
@@ -0,0 +1,63 @@
+import { render, RenderResult, screen } from '@testing-library/react'; //eslint-disable-line
+import React from 'react';
+
+import { documentOf } from '../../helpers';
+
+import Card from './Card';
+import { galaxy, title } from './Card.mock';
+
+const subtitle = '(1 Dec. 2021)';
+const toolbar = 'Toolbar content';
+const children = 'Dynamic content';
+
+describe('Component: <Card />', () => {
+	let rendered: RenderResult;
+
+	beforeEach(() => {
+		rendered = render(
+			<Card image={galaxy} title={title} subtitle={subtitle} toolbar={toolbar}>
+				{children}
+			</Card>
+		);
+	});
+
+	it('Should show a title', () => {
+		expect(screen.getByText(title)).toBeDefined();
+	});
+
+	it('Should show a subtitle', () => {
+		expect(screen.getByText(subtitle)).toBeDefined();
+	});
+
+	it('Should render toolbar content', () => {
+		expect(screen.getByText(toolbar)).toBeDefined();
+	});
+
+	it('Should render dynamic content', () => {
+		expect(screen.getByText(children)).toBeDefined();
+	});
+
+	it('Should render an image', () => {
+		const element = documentOf(rendered).getElementsByClassName('c-card__image-wrapper');
```

Comment:
Heb je `documentOf` nodig? Je kan rechtstreeks op de container een `querySelector` uitvoeren.

Reply:
Fixed in #14